[2024-10-16T09:54:37.790+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-03-12T00:00:00+00:00 [queued]>
[2024-10-16T09:54:37.807+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-03-12T00:00:00+00:00 [queued]>
[2024-10-16T09:54:37.809+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T09:54:37.810+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2024-10-16T09:54:37.811+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T09:54:37.835+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): insert_rate> on 2024-03-12 00:00:00+00:00
[2024-10-16T09:54:37.844+0000] {standard_task_runner.py:55} INFO - Started process 8241 to run task
[2024-10-16T09:54:37.853+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'exchange_rate_usd_kzt_dag', 'insert_rate', 'scheduled__2024-03-12T00:00:00+00:00', '--job-id', '1557', '--raw', '--subdir', 'DAGS_FOLDER/exchange_rates/__init__.py', '--cfg-path', '/tmp/tmpiwfil69s']
[2024-10-16T09:54:37.854+0000] {standard_task_runner.py:83} INFO - Job 1557: Subtask insert_rate
[2024-10-16T09:54:38.006+0000] {task_command.py:389} INFO - Running <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-03-12T00:00:00+00:00 [running]> on host 63ee4fa64e74
[2024-10-16T09:54:38.083+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:205: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2024-10-16T09:54:38.161+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=exchange_rate_usd_kzt_dag
AIRFLOW_CTX_TASK_ID=insert_rate
AIRFLOW_CTX_EXECUTION_DATE=2024-03-12T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-03-12T00:00:00+00:00
[2024-10-16T09:54:38.162+0000] {sql.py:253} INFO - Executing: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'KZT', 449.20604315, '2024-03-12')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;
[2024-10-16T09:54:38.173+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T09:54:38.460+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T09:54:38.469+0000] {sql.py:364} INFO - Running statement: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'KZT', 449.20604315, '2024-03-12')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;, parameters: None
[2024-10-16T09:54:38.473+0000] {sql.py:373} INFO - Rows affected: 1
[2024-10-16T09:54:38.490+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=exchange_rate_usd_kzt_dag, task_id=insert_rate, execution_date=20240312T000000, start_date=20241016T095437, end_date=20241016T095438
[2024-10-16T09:54:38.524+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2024-10-16T09:54:38.703+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-16T10:04:39.399+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-03-12T00:00:00+00:00 [queued]>
[2024-10-16T10:04:39.430+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-03-12T00:00:00+00:00 [queued]>
[2024-10-16T10:04:39.431+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T10:04:39.434+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2024-10-16T10:04:39.438+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T10:04:39.469+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): insert_rate> on 2024-03-12 00:00:00+00:00
[2024-10-16T10:04:39.477+0000] {standard_task_runner.py:55} INFO - Started process 10873 to run task
[2024-10-16T10:04:39.484+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'exchange_rate_usd_kzt_dag', 'insert_rate', 'scheduled__2024-03-12T00:00:00+00:00', '--job-id', '2309', '--raw', '--subdir', 'DAGS_FOLDER/exchange_rates/__init__.py', '--cfg-path', '/tmp/tmp34st9cfx']
[2024-10-16T10:04:39.486+0000] {standard_task_runner.py:83} INFO - Job 2309: Subtask insert_rate
[2024-10-16T10:04:39.653+0000] {task_command.py:389} INFO - Running <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-03-12T00:00:00+00:00 [running]> on host 63ee4fa64e74
[2024-10-16T10:04:39.800+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:205: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2024-10-16T10:04:39.948+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=exchange_rate_usd_kzt_dag
AIRFLOW_CTX_TASK_ID=insert_rate
AIRFLOW_CTX_EXECUTION_DATE=2024-03-12T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-03-12T00:00:00+00:00
[2024-10-16T10:04:39.951+0000] {sql.py:253} INFO - Executing: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'KZT', 449.20604315, '2024-03-12')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;
[2024-10-16T10:04:39.972+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T10:04:40.437+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T10:04:40.454+0000] {sql.py:364} INFO - Running statement: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'KZT', 449.20604315, '2024-03-12')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;, parameters: None
[2024-10-16T10:04:40.459+0000] {sql.py:373} INFO - Rows affected: 1
[2024-10-16T10:04:40.490+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=exchange_rate_usd_kzt_dag, task_id=insert_rate, execution_date=20240312T000000, start_date=20241016T100439, end_date=20241016T100440
[2024-10-16T10:04:40.825+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2024-10-16T10:04:40.885+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
